apps/frontend/src/libs/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,65 @@
+import { NumberInput } from "~/libs/components/components.js";
+import { useCallback, type usePagination } from "~/libs/hooks/hooks.js";
+
+import { NavigationButton } from "./libs/components/components.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	paginationSettings: ReturnType<typeof usePagination>;
+	totalItems: number;
+	totalPages: number;
+};
+
+const Pagination = ({
+	paginationSettings,
+	totalItems,
+	totalPages,
+}: Properties): React.JSX.Element => {
+	const {
+		goToEnd,
+		goToNext,
+		goToPrevious,
+		goToStart,
+		onPageSizeChange,
```

Comment:
Let's slightly rename them to follow conventions. We start with `on` only incoming property names, and we define the handlers in other places or in the hook return with handle
```suggestion
		handleGoToEnd,
		handleGoToNext,
		handleGoToPrevious,
		handleGoToStart,
		handlePageSizeChange,
```